### DISCUSSION TITLE
How Can I use Wire + UC?

### DISCUSSION THREAD

**Discussion by damidoug** [seleniumbase/SeleniumBase#2268] [General]
I'm trying to use, I need create an function and open it later in some threads to execute an action.

```
with Driver(uc=True, wire=True, headless=False) as driver:
        stealth(
            driver,
            languages=["en-US", "en"],
            vendor="Google Inc.",
            platform="Win32",
            webgl_vendor="Intel Inc.",
            renderer="Intel Iris OpenGL Engine",
            fix_hairline=True,
        )
        driver.get("https://nowsecure.nl/#relax")
        driver.sleep(4)
        for request in driver.requests:
            if request.response:
                print(
                    request.url,
                    request.response.status_code,
                    request.response.headers["Content-Type"],
                )
```

**Comment by mdmintz** [comment]
Duplicate of https://github.com/seleniumbase/SeleniumBase/issues/1774#issuecomment-1444399269, https://github.com/seleniumbase/SeleniumBase/issues/2032#issuecomment-1693931853, https://github.com/seleniumbase/SeleniumBase/issues/2151#issuecomment-1741234068, https://github.com/seleniumbase/SeleniumBase/issues/2243#issuecomment-17978647693, etc.

Due to unresolved issues with ``selenium-wire``: https://github.com/wkeeling/selenium-wire/search?q=undetected&type=issues, UC Mode and Wire Mode cannot be combined. Pick one or the other.
